client/app/src/main/java/com/microsoft/research/karya/ui/assistant/Assistant.kt

```diff
@@ -52,7 +52,7 @@ class Assistant(lifecycleOwner: LifecycleOwner) : LifecycleObserver {
         }
       }
     } else {
-      onErrorListener()
+      onCompletionListener()
```

Comment:
If assistant is not available, we should run both `onErrorListener()` and `onCompletionListener()` in order to simulate the MediaPlayer scenario.

Reply:
Or maybe have another listener to handle the case where no audio files are available? Since, we may show different messages as when assistant is not able to load when files are available and when no files are there

Reply:
I think we should probably only tell the user once if the audio files are not available. Add a TODO and we can look into this later.